### DISCUSSION TITLE
Is it OK if I distribute a docker image with a fully set-up Cross-Lib environment (including CC65)?

### DISCUSSION THREAD

**Discussion by Fabrizio-Caruso** [cc65/cc65#1574] [Q&A] [answered]
My Cross-Lib project depends on 
more than a dozen cross-compilers 
https://github.com/Fabrizio-Caruso/CROSS-LIB/blob/master/docs/COMPILERS.md, 
a native compiler, make, python and few other tools.

Is it OK if I create a docker image with a fully set-up Cross-Lib environment (including CC65) and distribute it freely?
Obviously I would include links to the CC65 and all the other compilers.
Cross-Lib is a totally free (for non-commercial use) and open source project.

This would make Cross-Lib so much easier to use for other people.

**Comment by mrdudz** [comment, marked as answer]
No problem with cc65 (which uses the zlib license)
